WEB-INF/applicationContext.xml

```diff
@@ -1097,6 +1097,15 @@
     <property name="daoFactory" ref="biospecimenDaoFactory"/>
   </bean>
 
+  <bean id="InstitutePrintRuleSvc" class="com.krishagni.catissueplus.core.common.service.impl.InstitutePrintRuleServiceImpl">
+    <property name="daoFactory" ref="biospecimenDaoFactory"/>
+    <property name="institutePrintRuleFactory" ref="institutePrintRuleFactory"/>
+  </bean>
+
+  <bean id="institutePrintRuleFactory" class="com.krishagni.catissueplus.core.common.domain.factory.impl.InstitutePrintRuleFactoryImpl">
+    <property name="daoFactory" ref="biospecimenDaoFactory"/>
```

Comment:
Why is this named institute print rule? The print rule has nothing to do with institute.